admin/client/EditorCustomizeTab.tsx

```diff
@@ -37,30 +49,104 @@ class ColorSchemeSelector extends React.Component<{ chart: ChartConfig }> {
         this.props.chart.props.invertColorScheme = value || undefined
     }
 
+    createLinearGradient(colorScheme: ColorScheme, count: number) {
+        let colors = colorScheme.getColors(count)
+        if (this.props.chart.props.invertColorScheme) colors = colors.reverse()
+
+        const step = 100 / count
+        const gradientEntries = colors.map(
+            (color, i) => `${color} ${i * step}%, ${color} ${(i + 1) * step}%`
+        )
+
+        return `linear-gradient(90deg, ${gradientEntries.join(", ")})`
+    }
+
+    @action.bound formatOptionLabel(option: ColorSchemeOption) {
+        return (
+            <div
+                style={{
+                    display: "flex",
+                    justifyContent: "space-between",
+                    alignItems: "center"
+                }}
+            >
+                <div>{option.label}</div>
+                <span
+                    style={{
+                        backgroundImage: this.createLinearGradient(
+                            option.colorScheme,
+                            6
+                        ),
+                        width: "6rem",
+                        height: "1.25rem",
+                        border: "1px solid #aaa"
+                    }}
+                />
+            </div>
+        )
+    }
+
     render() {
         const { chart } = this.props
 
-        const availableColorSchemes = keysOf(ColorSchemes)
-        const colorSchemeLabels = availableColorSchemes.map(
-            scheme => (ColorSchemes[scheme] as ColorScheme).name
+        const defaultColorScheme = {
+            value: "default",
+            label: "Default",
+
+            // This is the default color in DiscreteBarTransform
+            colorScheme: new ColorScheme(
+                "DiscreteBarChart default",
+                [["#2e5778", "#2e5778", "#2e5778"]],
+                false
```

Comment:
This is ugly: I'm creating a new ColorScheme containing just the color `#2e5778`, which is what `DiscreteBarTransform` currently uses as its default.
I want to be able to also show a preview of the "default" theme, but to get that to work I need a ColorScheme.
Maybe we can move this code to `ColorSchemes`??

Reply:
This got better with the latest commit, where one `additionalColorScheme` is provided as a `ColorSchemeOption` object, with no need to actually create a `ColorScheme` for it.
Instead, its `gradient` is specified directly.